source/core/TDCodecVia.cpp

```diff
@@ -1592,7 +1592,7 @@ void TDViaParser::ParseVirtualInstrument(TypeRef viType, void* pData)
     // The clumps code will be loaded once the module is finalized.
 }
 //------------------------------------------------------------
-void TDViaParser::FinalizeVILoad(VirtualInstrument* vi, EventLog* pLog)
+void TDViaParser::FinalizeVILoad(VirtualInstrument* vi, EventLog* pLog, Boolean idIDE)
```

Comment:
Vireo code shouldn't have terminology such as IDE. Name this as isDebuggingEnabled as well.